pages/signup.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import { Button, Form } from 'react-bootstrap';
+import { useAuth } from '../context/AuthContext';
+
+const Signup = () => {
+  //  const { user, login } = useAuth();
+  const { signup } = useAuth();
+  //   console.log('User:' + user);
+  const [data, setData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleSignup = async (e: any) => {
+    e.preventDefault();
+
+    try {
+      await signup(data.email, data.password);
```

Comment:
If signup is successful we should bring the user back to the previous page they were on with them now logged in

Reply:
Could talk about this later easy fix